src/applications/edu-benefits/tests/1990e/01-all-fields.e2e.spec.js

```diff
@@ -16,6 +16,10 @@ module.exports = E2eHelpers.createE2eTest(client => {
       }/education/apply-for-education-benefits/application/1990E`,
     )
     .waitForElementVisible('body', Timeouts.normal)
+    // uncomment when in production
+    // .waitForElementVisible('.wizard-container', Timeouts.normal)
+    // .waitForElementVisible('.skip-wizard-link', Timeouts.normal)
+    // .click('.skip-wizard-link')
```

Comment:
why is this commented out?

Reply:
The previous PR failed e2e because of these lines and the classes are correct. Not sure if these tests  run in a production environment but if they do, it will fail since this is not going to production yet.

Reply:
This doesn't make a lot of sense to me - if these fail when not in a prod environment and you need a PR to be merged for something to get into a prod environment, these tests will never pass..  I will comment on the line of code that I think is the culprit

Reply:
These tests will fail if on prod. The wizard isn't be shown in prod yet.